docs/transpile/custom-transpiler-pass.ipynb

```diff
@@ -0,0 +1,352 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Writing your own Pass\n",
+    "\n",
+    "In the [Transpiler stages](transpiler-stages) topic we went over the default behavior of the transpiler and the various stages that compose a transpilation pipeline.  Qiskit allows you to create custom transpilation workflows using: the `PassManager` and `StagedPassMager` objects, writing your own bespoke transpiler passes, and  Here we will take a more careful look at the `PassManager` and `StagedPassManager` objects to customize which passes are executed at what stage and walk through a few examples.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## DAG Representation\n",
+    "\n",
+    "\n",
+    "However, before we start it is important to first introduce the internal representation of quantum circuits in Qiskit, the *Directed Acyclic Graph* or **DAG**.  If you want to follow along here, you will need to install the `pydot` library and the `graphivz` library for the DAG plotting functions.  Use the python package manager of your choice (such as `pip` or `conda`) to install `pydot` and your system's native package manager (e.g. `apt`, `brew`, `yum`, `dnf`, etc.) for `graphivz`.\n",
```

Comment:
FWIW, pydot isn't a requirement for the dag drawer anymore (not since terra 0.21.0), it looks like the docstring for `DAGCircuit.draw()` needs to be updated because the PR that updated to directly invoke graphviz via rustworkx only touched the `dag_drawer` function and not it's alias method on `DAGCircuit`. 

The only things using pydot directly is the pass manager visualization.

Reply:
Removed `pydot` as a requirement.

Reply:
I just pushed up: https://github.com/Qiskit/qiskit/pull/11337 to fix the method documentation